tests/AbstractErrorHandlerTest.php

```diff
@@ -44,6 +45,34 @@ public function testConstructor()
         }
     }
 
+    /**
+     * @dataProvider errorReportingConstantProvider
+     */
+    public function testHandleErrorShouldNotCapture(bool $expectedToCapture, int $captureAt, int $errorReporting)
+    {
+        if (!$expectedToCapture) {
+            $this->client->expects($this->never())
+                ->method('capture');
+        }
+        $errorHandler = $this->createErrorHandler($this->client);
+        $errorHandler->captureAt($captureAt, true);
+        $prevErrorReporting = error_reporting($errorReporting);
```

Comment:
Please put a blank line above this one to improve readability (I tend to leave a blank line above/below an assignment line if the previous one is not an assignment line too)